testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -648,7 +652,7 @@ fn invalid_tx(
     let seed: [u8; 32] = OsRng.gen();
     let mut rng = StdRng::from_seed(seed);
     let mut invalid_account = gen_random_account(&mut rng);
```

Comment:
You should look at removing the gen_random_account function and just use `LocalAccount::generate` directly.